CIP-1775/README.md

```diff
@@ -0,0 +1,108 @@
+---
+CIP: 1775
+Title: Delegated Spending Authorizations
+Authors: Ivan Mikushin <sorrel_cantos.0@icloud.com>
+Comments-URI:
+Status: Proposed
+Type: Standards Track
+Created: 2022-11-17
+License: CC-BY-4.0
+---
+
+# Delegated Spending Authorizations
+
+"Give me custody, or give me death!" - [@ArmySpies](https://twitter.com/ArmySpies/status/1592586212816355328)
+
+## Abstract
+
+We propose to allow users to delegate authorized spending of outputs in their wallets to smart contracts or other users, without sending said outputs out of their wallets.
+
+## Motivation
+
+In order to use any dApp on Cardano (e.g. a decentralized exchange or an NFT marketplace), a user has to **deposit tokens** to a script address, thereby **giving up custody** of the tokens.
+
+This can be problematic for a number of reasons.
+
+- The user loses control of the assets they send out to a smart-contract, unless the smart contract is specifically designed to allow for such control (e.g. giving a "Cancel trade" or "De-list NFT" button in the UI).
+- Assets sent out from a user's wallet:
+  - are no longer  **delegated** to the stake pool of the user's choice.
+  - cannot participate in **governance voting** on behalf of the user.
+- Any output, accidentally sent to a script address without a datum is effectively lost now (because of how transaction validation scripts are invoked).
+- Requiring users to deposit assets into a dapp may subject the dapp's owners to regulatory scrutiny.
+
+We believe, this is not strictly necessary and would like a way to allow users to delegate spending authorization for some outputs **within** their wallets to other users (smart contracts, multisigs, wallet addresses).
+
+### Use Cases
+
+Instead of sending tokens to the dapp smart-contract, the user creates an output in their own wallet that delegates spending authorization to a smart-contract (or other entity).
```

Comment:
We could look into extending native scripts: 
- add `datum_option` to any Plutus script address referred by the native script,
- include the native script into the script context passed to the referred Plutus scripts.

This doesn't address staking and voting. Integration with sidechains (and Hydra) will not be as seamless as it could be with the proposed CIP.